docs/3___model_exchange.adoc

```diff
@@ -1,18 +1,44 @@
 == FMI for Model Exchange [[fmi-for-model-exchange]]
 
-This chapter contains the interface description to access the equations of a dynamic system from a C program.
-A schematic view of a model in FMI for Model Exchange format is shown in <<figure-model-exchange-data-flow>>:
+FMI for Model Exchange provides a standardized interface for solving ordinary differential equations with events (abbreviated as "hybrid ODE").
+In contrast to FMI for Co-Simulation, a Model Exchange FMU implements the model algorithm only, without the required solution method.
 
-.Data flow between the environment and an FMU for Model Exchange
+The importer controls the data exchange and the synchronization between FMUs.
+
+The solver algorithm itself is not part of the FMI standard.
+
+The solver algorithm is responsible for:
+
+* advancing the overall simulation time,
+* exchange input and output data,
+* computation of continuous state variables by time integration,
+* triggering of input clocks, and
+* handling events.
+
+In contrast to Co-Simulation, the FMU is evaluated only at a specific time instant and not from one communication point to the next.
+A Model Exchange FMU might consist of discrete-time equations only, for example, describing a sampled-data controller.
+
+A schematic view of an FMU for Model Exchange is shown in <<figure-model-exchange-data-flow>>:
+
+.Data flow between the solver of the importer and the FMU for Model Exchange
 [#figure-model-exchange-data-flow]
 image::images/model-exchange-data-flow.svg[width=80%, align="center"]
 
-[blue]*Blue* arrows: Information provided by the FMU. +
-[red]*Red* arrows : Information provided to the FMU. +
-latexmath:[\mathbf{v}_{\mathit{start}}], latexmath:[\mathbf{u}], latexmath:[\mathbf{y}], and latexmath:[\mathbf{w}], are of a numeric type or string;
-latexmath:[\mathbf{t}], latexmath:[\mathbf{x}_c], latexmath:[\mathbf{z}] are of floating point type.
+FMI for Model Exchange enables the following features:
+
+* fixed or variable step, explicit or implicit integration algorithms,
+
+* exact localization of <<state-event,state events>> using <<fmi3GetEventIndicators,event indicators>>,
+
+* dedicated <<InitializationMode>>, which allows computation of consistent initial conditions over <<algebraic-loops,algebraic loops>>,
+
+* handling of <<clock,clocks>> and <<EventMode,events>>,
+
+* solution of algebraic loops involving inputs and output variables of this FMU (see <<algebraic-loops,algebraic loops>>),
+
+* selective computation of FMU variables required by the solver algorithm, e.g. outputs, event indicators, derivatives (see <<selectiv-computation>>),
+
+//TODO: Christian: is this the correct formulation?
```

Comment:
fine for me.

Reply:
Can I remove the TODO?

Reply:
@chrbertsch : There are two TODOs for you in the text (search for "TODO: Christian:"), can you please look at both and either remove or fix?